### DISCUSSION TITLE
Windows 11 as a guest in VirtualBox

### DISCUSSION THREAD

**Discussion by spflanze** [openzfs/zfs#13892] [Q&A]
I have installed, and have running, an Ubuntu 22.04.1 OS on an encrypted, and RAIDed, ZFS file system. The instructions I followed are given here::
https://openzfs.github.io/openzfs-docs/Getting%20Started/Ubuntu/Ubuntu%2022.04%20Root%20on%20ZFS.html
In these instructions there is this:
> Do not use these instructions for dual-booting.

Does that also include attempting to run Windows 11 as a guest in VirtualBox? I have trouble attempting to do that.

**Comment by rincebrain** [comment]
Running a VM is not the same as dual-booting, no.
